Won't compile for wasm32-unknown-unknown

`bevy_rapier` is defined in my `Cargo.toml` as follows:

```
bevy_rapier3d = { version = "0.5.0", features = ["wasm-bindgen"] }
```

When I'm trying to compile my project for `wasm32-unknown-unknown`, it fails with the following errors:

```
error[E0425]: cannot find function `allocation_granularity` in this scope
 --> /home/mvlabat/.cargo/registry/src/github.com-1ecc6299db9ec823/slice-deque-0.3.0/src/mirrored/buffer.rs:7:14
  |
7 |     let ag = allocation_granularity();
  |              ^^^^^^^^^^^^^^^^^^^^^^ not found in this scope

error[E0425]: cannot find function `allocation_granularity` in this scope
   --> /home/mvlabat/.cargo/registry/src/github.com-1ecc6299db9ec823/slice-deque-0.3.0/src/mirrored/buffer.rs:112:15
    |
112 |             * allocation_granularity();
    |               ^^^^^^^^^^^^^^^^^^^^^^ not found in this scope

error[E0425]: cannot find function `allocation_granularity` in this scope
   --> /home/mvlabat/.cargo/registry/src/github.com-1ecc6299db9ec823/slice-deque-0.3.0/src/mirrored/buffer.rs:135:41
    |
135 |         assert!(mem::align_of::<T>() <= allocation_granularity());
    |                                         ^^^^^^^^^^^^^^^^^^^^^^ not found in this scope

error[E0425]: cannot find function `allocation_granularity` in this scope
   --> /home/mvlabat/.cargo/registry/src/github.com-1ecc6299db9ec823/slice-deque-0.3.0/src/mirrored/buffer.rs:148:36
    |
148 |         debug_assert!(alloc_size % allocation_granularity() == 0);
    |                                    ^^^^^^^^^^^^^^^^^^^^^^ not found in this scope

error[E0425]: cannot find function `allocate_mirrored` in this scope
   --> /home/mvlabat/.cargo/registry/src/github.com-1ecc6299db9ec823/slice-deque-0.3.0/src/mirrored/buffer.rs:151:19
    |
151 |         let ptr = allocate_mirrored(alloc_size)?;
    |                   ^^^^^^^^^^^^^^^^^ not found in this scope

error[E0425]: cannot find function `deallocate_mirrored` in this scope
   --> /home/mvlabat/.cargo/registry/src/github.com-1ecc6299db9ec823/slice-deque-0.3.0/src/mirrored/buffer.rs:172:18
    |
172 |         unsafe { deallocate_mirrored(first_half_ptr, buffer_size_in_bytes) };
    |                  ^^^^^^^^^^^^^^^^^^^ not found in this scope

error: aborting due to 9 previous errors
```

This might be caused by the fact that `bevy_rapier` requires `bevy` with a default set of features, while some of them don't compile for `wasm32-unknown-unknown`. I believe this can be fixed with specifying the minimal set of features for `bevy`, which are needed for `bevy_rapier` to work.

--- a/bevy_rapier2d/Cargo.toml
+++ b/bevy_rapier2d/Cargo.toml
@@ -28,7 +28,7 @@ serde-serialize = [ "rapier2d/serde-serialize" ]
 enhanced-determinism = [ "rapier2d/enhanced-determinism" ]
 
 [dependencies]
-bevy = "0.3"
+bevy = { version = "0.3", default-features = false, features = ["render"] }
 nalgebra = "0.23"
 rapier2d = "0.3"
 concurrent-queue = "1"

--- a/bevy_rapier3d/Cargo.toml
+++ b/bevy_rapier3d/Cargo.toml
@@ -28,7 +28,7 @@ serde-serialize = [ "rapier3d/serde-serialize" ]
 enhanced-determinism = [ "rapier3d/enhanced-determinism" ]
 
 [dependencies]
-bevy = "0.3"
+bevy = { version = "0.3", default-features = false, features = ["render"] }
 nalgebra = "0.23"
 rapier3d = "0.3"
 concurrent-queue = "1"

